build-binaries.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+VERSION=v0.1.0
+
+ARCH=$(uname -m)
+[ $ARCH = x86_64 ] && ARCH=amd64
+[ $ARCH = aarch64 ] && ARCH=arm64
```

Comment:
Is it possible to build for a platform other than the host architecture by providing a --platform argument to docker build? For example, if I'm on an Ubuntu x86 machine could I build linux/arm64 by calling `docker build --platform linux/arm64 ... `?

If the answer is yes, then could we add a flag to this script to allow for that? Default behavior could be to use the host architecture.

Reply:
It's not quite so simple. We can extend this for other platforms, much like is done in the github actions, but it'll take a bit of work and I'm doing to defer it beyond this PR.

https://rust-lang.github.io/rustup/cross-compilation.html